src/main/java/io/kestra/plugin/fs/ssh/Command.java

```diff
@@ -0,0 +1,103 @@
+package io.kestra.plugin.fs.ssh;
+
+import com.jcraft.jsch.ChannelExec;
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.Session;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.models.tasks.Task;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.fs.vfs.AbstractVfsInterface;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+import java.io.ByteArrayOutputStream;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Send a command to a remote server using SSH."
+)
+@Plugin(
+    examples = {
+        @Example(
+            code = {
+                "host: localhost",
+                "port: 22",
+                "username: foo",
+                "password: pass",
+                "command: ls",
+            }
+        )
+    }
+)
+public class Command extends Task implements AbstractVfsInterface, RunnableTask<VoidOutput> {
+    private static final long SLEEP_DELAY = 25L;
```

Comment:
```suggestion
    private static final long SLEEP_DELAY = 1L;
```

Reply:
Well, I can lower it but 1ms seems to low.
An SSH command will  imply a network round trip, a remote session on the server be established, and the command itself. So 10ms or 5ms min seems OK.

Reply:
ok I was thinking about second don't know why :face_exhaling:

Reply:
:)

I'll rename the const to add `MS` at the end ... it's a best practice to always add the unit in the name of such parameter.